src/query-builder/RelationIdLoader.ts

```diff
@@ -636,7 +637,7 @@ export class RelationIdLoader {
                 undefined,
                 primaryColumn.entityMetadata.name +
                     "_" +
-                    relation.inverseRelation!.propertyPath.replace(".", "_") +
+                    originalRelation.propertyPath.replace(".", "_") +
```

Comment:
Since this is not part of the revert, what's happening here? :eyes:

Reply:
A test was failing because the value was undefined.